CHANGES.md

```diff
@@ -1,9 +1,138 @@
+- [#4219](https://github.com/gap-system/gap/pull/4219) Fixed `IntermediateGroup`: For large indices, it might have returned `fail` although subgroups exist, instead of issuing an error message or warning about a hard calculation
+- [#4232](https://github.com/gap-system/gap/pull/4232) Iterators for lists: fixed `IsDoneIterator`, improved the performance
+- [#4239](https://github.com/gap-system/gap/pull/4239) Added missing comparison w.r.t. equality of something and an object with memory (`IsObjWithMemory`)
+- [#4245](https://github.com/gap-system/gap/pull/4245) Free (associative or Lie) algebras: admit non-fields as left acting domains, fixed the zero-dimensional case
+
+-->
+
+## GAP 4.11.1 (February 2021)
+
+### Fixed bugs that could lead to incorrect results
+
+- [#4178](https://github.com/gap-system/gap/pull/4178) Fixed bugs in `RestrictedPerm` with second argument a range
+
+### Fixed bugs that could lead to crashes
+
+- [#3965](https://github.com/gap-system/gap/pull/3965) Fix potential garbage collector crashes on 64bit ARM systems
+- [#4076](https://github.com/gap-system/gap/pull/4076) Fix an infinite loop in `BoundedRefinementEANormalSeries` if large factors could not be refined, fix protected option of `IsomorphismSimplifiedFpGroup`, improve documentation of `IsAutomorphismGroup`
+
+### Fixed bugs that could lead to error messages
+
+- [#3980](https://github.com/gap-system/gap/pull/3980) Fixed `Gcd` for rational polynomials
+
+### Other fixed bugs
+
+- [#3963](https://github.com/gap-system/gap/pull/3963) Provide automatic compression/decompression of filenames ending `.gz` (as is claimed for example in the documentation of `InputTextFile`)
+- [#3944](https://github.com/gap-system/gap/pull/3944) The error checking in `PartialPerm` has been corrected such that invalid inputs (numbers < 1) are detected
+- [#4006](https://github.com/gap-system/gap/pull/4006) Fix `gac` to ensure that binaries it creates on Linux can load and run, even if a GAP package with a compiled kernel extension (such as `IO`) is present
+
+### Improved and extended functionality
 
-These changes are also listed on the
-[Wiki page](https://github.com/gap-system/GAP/wiki/gap-4.11-release-notes)
+- [#3790](https://github.com/gap-system/gap/pull/3790) Add further information to the library of simple groups
+- [#3840](https://github.com/gap-system/gap/pull/3840) Speed up garbage collection
+
+### Packages
+
+- [#4016](https://github.com/gap-system/gap/pull/4016) Improved `etc/Makefile.gappkg` (used by GAP packages that want to build a simple GAP kernel extension)
+
+### Fixes/improvements in the experimental way to allow 3rd party code to link GAP as a library (libgap)
+
+- [#4081](https://github.com/gap-system/gap/pull/4081) Enhance `GAP_ValueGlobalVariable` to supported automatic variables (see `DeclareAutoreadableVariables`)
```

Comment:
I am afraid we have a late addition:
```suggestion
- [#4081](https://github.com/gap-system/gap/pull/4081) Enhance `GAP_ValueGlobalVariable` to supported automatic variables (see `DeclareAutoreadableVariables`)
- [#4258](https://github.com/gap-system/gap/pull/4258) Fixed `GAP_Enter` macro so that GAP's recursion depth counter is saved/restored. Without this, if too many GAP errors occurred during runtime a segmentation fault could occur in the program using libgap
```